src/backend/utils/activity/pgstat_relation.c

```diff
@@ -26,6 +26,9 @@
 #include "utils/rel.h"
 #include "utils/timestamp.h"
 
+#include "parser/parser.h"
+#include "utils/queryenvironment.h"
```

Comment:
do we need this header?

Reply:
yes. sql_dialect and RELPERSISTENCE_TEMP below